cli/serverless/golang/templates/main.tmpl

```diff
@@ -4,6 +4,12 @@ func InputSchema() any {
 }
 {{end}}
 
+{{if not .WithDataTags}}
```

Comment:
`not`? typo

Reply:
This is `run` template. It checks whether the user's code has defined`DataTags()`. If not defined, it will automatically add an empty one for the user

If do not do this, the compilation will fail.

Reply:
👌